ci/glci/alicloud.py

```diff
@@ -0,0 +1,219 @@
+import oss2
+import sys
+import os
+import datetime
+import tempfile
+import json
+import time
+import logging
+import dataclasses
+import glci.model
+from aliyunsdkecs.request.v20140526 import ImportImageRequest
+from aliyunsdkecs.request.v20140526 import DescribeImagesRequest
+from aliyunsdkecs.request.v20140526 import DescribeRegionsRequest
+from aliyunsdkecs.request.v20140526 import CopyImageRequest
+from aliyunsdkcore.client import AcsClient
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
+TIME_OUT = 30 * 60  # in seconds, 30 Minutes
+
+
+class AlicloudImageStatus:
```

Comment:
nit: rather use `enum.Enum` as base class (this should be a one-line-change) -->
```
class AlicloudImageStatus(enum.Enum):
...
```